src/main/java/views/add_edit_category_views/AddCategoryV.java

```diff
@@ -0,0 +1,70 @@
+package views.add_edit_category_views;
+
+import entities.SessionStorage;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+/**
+ * A view class for the Add_Category_view that extends Component class and implements ActionListener interface.
+ * Creates a new controller that produces a Category_OD object.
+ */
+
+public class AddCategoryV extends Component implements ActionListener {
+    CategoryC controller;
+    JTextField name_input;
+    JTextField budget_input;
+    public AddCategoryV(CategoryC controller,int monthID, SessionStorage curr_session) {
+        /**
+         * Builds Add_Category_View.
+         */
+        JLabel name_label = new JLabel("Category Name:");
+        this.name_input = new JTextField(15);
+        JLabel value_label = new JLabel("Category Budget:");
+        this.budget_input = new JTextField(15);
+        JButton submit = new JButton("Submit");
+        submit.setSize(30,10);
+
+        JFrame frame = new JFrame();
+        JPanel panel = new JPanel();
+        panel.setBorder(BorderFactory.createEmptyBorder(50, 30, 50, 30));
+        panel.setLayout(new GridLayout(0,1));
+        JPanel panell = new JPanel();
+        panell.setBorder(BorderFactory.createEmptyBorder(50, 30, 50, 30));
+        panell.setLayout(new GridLayout(0,1));
+
+        frame.add(panel, BorderLayout.CENTER);
+        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        frame.setTitle("Add New Category");
+        frame.setSize(300,500);
+        frame.pack();
+        frame.setVisible(true);
```

Comment:
pack() and setVisible(true) after you have added all components to the frame. Also, you need to use .addActionListener(this) on submit button.